policy/modules/system/sysnetwork.if

```diff
@@ -364,6 +364,31 @@ interface(`sysnet_read_config',`
 	')
 ')
 
+#######################################
+## <summary>
+##     Map network config files.
+## </summary>
+## <desc>
+##     <p>
+##     Allow the specified domain to mmap the
+##     general network configuration files.
+##     </p>
+## </desc>
+## <param name="domain">
+##     <summary>
+##     Domain allowed access.
+##     </summary>
+## </param>
+#
+interface(`sysnet_map_config',`
```

Comment:
I think that mmap is used used consistently in refpolicy. So i guess something like `sysnet_mmap_config_files()` would be better